barakadanny/Investment-app
# Investment App

## Pull Request

### Description

This pull request introduces several improvements and new features to the Investment Calculator project, enhancing its functionality and user experience.

### Changes Made

- Implemented data entry reset functionality in the Investment Form.
- Added a message when there's no data available in the Investment Calculator.
- Updated the InvestmentsItems component to display calculated investment data.
- Improved the overall structure and user interface of the application.
- Added a new feature to calculate and display investment results based on user inputs.

### How to Test

To test these changes, follow these steps:

1. Open the Investment Calculator application.
2. Input investment data.
3. Click the "Calculate" button to see the investment results.
4. Use the "Reset" button to clear the input fields.
5. Verify that a message is displayed when there's no data available.

### Related PRs

- None.

### Additional Information

These changes enhance the user experience and make the Investment Calculator more user-friendly. The updated components and features improve the overall functionality and user interface.

### Reviewers

@barakadanny
